.github/workflows/deploy-staging.yml

```diff
@@ -16,9 +16,13 @@ jobs:
         run: |
           echo REACT_APP_GITHUB_CLIENT_ID=$GITHUB_CLIENT_ID >> .env
           echo REACT_APP_API_URL=$API_URL >> .env
+          echo REACT_APP_ENVIRONMENT=$ENVIRONMENT >> .env
+          echo REACT_APP_ROLLBAR_ACCESS_TOKEN=$ROLLBAR_ACCESS_TOKEN >> .env
```

Comment:
jak chcesz wszystkie na raz wstrzyknąć, to może `envsubst`a użyć, tylko wtedy muszą się nazwy zgadzać ;)